rails-snow-queen/app/assets/javascripts/map.js

```diff
@@ -7,6 +7,7 @@ class Map {
     this.marker = null;
     this.geocodedAddress = null;
     this.polygons = [];
+    this.polygonsJSON = null;
```

Comment:
Not used

Reply:
right, i moved it to a different class and forgot to remove it here